File: lab1/time_logic.py
total_seconds = int(input("Enter total seconds"))
hours = total_seconds // 3600
remaining = total_seconds % 3600

minutes = remaining // 60
seconds = remaining % 60

print(str(total_seconds) + " seconds is " + str(hours) + " hours, " + str(minutes) + " minutes, and " + str(seconds) + " seconds." )

File: lab1/time_logic.cpp

#include <iostream>
using namespace std;
int main() {
    int total_seconds;
    cout<<"Enter total seconds: ";
    cin>> total_seconds;

    int hours = total_seconds / 3600;
    int remaining = total_seconds % 3600;
    int minutes = remaining / 60;
    int seconds = remaining % 60;

    cout<<total_seconds<<" second is "<<hours<<" hours, "<<minutes<<" minutes, and "<< seconds<<" seconds."<<endl;
    return 0;

}